dali/operators/reader/video_reader_resize_op.h

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_READER_VIDEO_READER_RESIZE_OP_H_
+#define DALI_OPERATORS_READER_VIDEO_READER_RESIZE_OP_H_
+
+#include <string>
+#include <vector>
+
+#include "dali/operators/reader/loader/video_loader.h"
+#include "dali/operators/reader/reader_op.h"
+#include "dali/operators/reader/video_reader_op.h"
+#include "dali/operators/image/resize/resize.h"
+
+namespace dali {
+
+class VideoReaderResize : public VideoReader, protected ResizeAttr, protected ResizeBase {
```

Comment:
Inheritance or composition?
I don't think ti should inherit from ResizeAttr, not sure about ResizeBase either.

Reply:
This is how `Resize` is built, so I wanted to follow this pattern. If we want to change, let's change both, in separate PR maybe?

Reply:
Sure.